test/integration/PublishEditConsume.test.ts

```diff
@@ -78,7 +78,8 @@ const ifpsFile: Files = {
   files: [
     {
       type: 'ipfs',
-      hash: 'QmdMBw956S3i2H2ioS9cERrtxoLJuSsfjzCvkqoDgUa2xm'
+      hash: 'QmRhsp7eghZtW4PktPC2wAHdKoy2LiF1n6UXMKmAhqQJUA'
+      // not exists anymore? 'QmdMBw956S3i2H2ioS9cERrtxoLJuSsfjzCvkqoDgUa2xm'
```

Comment:
Is this the only test where we were using `QmdMBw956S3i2H2ioS9cERrtxoLJuSsfjzCvkqoDgUa2xm`?

Reply:
i believe so. btw, i picked the new hash from the ips tests on ocean node (we had similar issues there with ipfs tests some time ago).. i might be wrong but i have the impression these ipfs files get deleted  after a while